master/buildbot/status/web/logs.py

```diff
@@ -117,41 +122,53 @@ def render_GET(self, req):
         else:
             req.setHeader("Cache-Control", "no-cache")
 
-        if not self.asText:
-            self.template = req.site.buildbot_service.templates.get_template("logs.html")
-            self.chunk_template = req.site.buildbot_service.templates.get_template("log_chunk.html")
-            builder = self.original.step.build.builder
-            build_id = self.original.step.build.number
-            url_dict = self.original.master.status.getURLForBuild(builder.getName(), build_id)
+        # If plaintext is requested just return the content of the logfile
+        if self.asText:
+            with_headers = "_with_headers" if self.withHeaders else ""
+            base_name = self.original.step.getName() + "_" + self.original.getName() + with_headers
+            base_name = base_name.replace(" ", "_") + ".txt"
+            req.setHeader("Content-Disposition", "inline; filename =\"" + base_name + "\"")
```

Comment:
Yes, but as long as copy/pasting from the logs doesn't work as intended in Chrome, then I think it makes sense to be able to access the raw file in the browser especially since this is a temporary thing. But if you don't think that's a good idea I'll change it.